### PR TITLE
Add dataset names to paths in sql/

### LINKED ISSUE #189: Move queries into dataset-named folders in sql/

### PR DESCRIPTION
**Pull request by relud**
fixes #189

### CODE REVIEW

File: tests/test_generated.py [hunk 1]
```diff
@@ -10,15 +10,20 @@
...
 @pytest.fixture(scope="session")
 def bq():
     return bigquery.Client()
 
 
-@pytest.fixture(params=list(generate_tests()))
+@pytest.fixture(params=list(generated_tests.keys()))
 def generated_test(request):
-    return request.param
+    return generated_tests[request.param]
```
Comment (relud) [inline tests/test_generated.py]: this makes `pytest` report the generated test path in test names for easier debugging, which I needed for verifying that all the tests were still being run

File: script/generate_sql [hunk 2]
```diff
@@ -51,7 +51,13 @@ def main():
...
+    for root, dirs, files in os.walk(args.sql_dir):
+        sql_files = [filename for filename in files if filename.endswith(".sql")]
+
+        basename = root.replace(args.sql_dir, args.destination, 1)
+        if not os.path.exists(basename) and (dirs or sql_files):
+            os.mkdir(basename)
+
         for filename in sql_files:
             # make sure only SQL files are considered
             if not filename.endswith(".sql"):
```
Comment (jklukas) [inline script/generate_sql]: We now check this condition earlier, so can remove it here.

**Comment by jklukas** [comment]
I assume this will require a telemetry-airflow PR to match the new paths?

**Comment by relud** [comment]
yes

**Pull request merged**